dali/ft/filecopy.cpp

```diff
@@ -2662,6 +2662,16 @@ void FileSprayer::checkFilePath(RemoteFilename & filename)
             Owned<IConstDropZoneInfo> dropZone = env->getDropZoneByAddressPath(netaddress.str(), pfilePath);
             if (!dropZone)
                 LOG(MCdebugInfo, unknownJob, "No matching drop zone path to file path: '%s'", filePath.str());
+            else
+            {
+                SCMStringBuffer dropZoneName;
+                dropZone->getName(dropZoneName);
+                LOG(MCdebugInfo, unknownJob, "Drop zone path '%s' is %svisible in ECLWatch."
```

Comment:
One line per (target/source) file in DFU server logs. My aim was to create same diagnostic log as we create if the target isn't in a valid dropzone.